Triphouse-project/index.html

```diff
@@ -1,33 +1,35 @@
 <!DOCTYPE html>
```

Comment:
![2021-04-22_21h50_13](https://user-images.githubusercontent.com/19935417/115770069-e668ca00-a3b4-11eb-8f31-dbd5bd1074d8.png)
Wrap and add to wrapper max-width: 1140px